natlas-server/entrypoint.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/sh
 
-mkdir -p /data/{db,media}
+mkdir -p /data/{db,media} logs
```

Comment:
Nitpicking now but `logs` should be inside the {} expansion, since there are already other things there and now it doesn't really adhere to any particular style. I am partial to putting it inside the expansion instead of passing many arguments directly to mkdir without fully qualified paths. The `/data/{db,media,logs}` would be more understandable than `db media logs`, for instance.

Reply:
Logs is actually outside the data dir and in the WORKDIR of the docker container.

Reply:
Oh right, that's because I'm a bad programmer haha. Logs should be put somewhere persistent but because the server isn't _actually_ logging and is instead just spitting out a few print statements into text files when certain events happen, they aren't centrally managed.

For now this is probably okay, but the logs file should eventually be put into the data dir so that they persist beyond container restart/upgrade. It's up to you whether or not you want to take that on.

Reply:
Its simple. I could probably switch this to a symlink instead.